.github/workflows/test_action.yml

```diff
@@ -14,9 +16,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - uses: arthurpalves/coherent-swift@0.5.10
+    - uses: arthurpalves/coherent-swift@0.5.5
```

Comment:
As this is intended to make sure the Github Action is working based on current changes, it should point to `@main` instead of `@X.Y.Z`.

Thanks for spotting this anyway.